src/lhttpc_manager.erl

```diff
@@ -271,7 +271,7 @@ init(Options) ->
         true ->
             % Make sure that the ssl random number generator is seeded
             % This was new in R13 (ssl-3.10.1 in R13B vs. ssl-3.10.0 in R12B-5)
-            apply(ssl, seed, [crypto:rand_bytes(255)]);
+            apply(ssl, seed, [crypto:strong_rand_bytes(255)]);
```

Comment:
In other words, this code won't even run on modern OTP versions (good thing it checks whether the seed function is exported, which according to the comment was a new thing back in R13.)